cloudferrylib/utils/networks.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2016 Mirantis Inc.
```

Comment:
I'd recommend keeping `networks.py` and `floatingip.py` in `cloudferrylib.os.network`